### DISCUSSION TITLE
gitbook cannot synchornize with github / gitlab 2 days ago

### DISCUSSION THREAD

**Discussion by junxu-ai** [GitbookIO/community#742] [3. Git Sync]
1. My last sync with github was 2 days ago. after that gitbook stops the actual sync with github. I use my same gmail account to log in github and gitbook
2. I tried several methods from google, but neither worked. For the latest one, i log out and log in, and tried to sync the contents in gitbook to a new repos in github. it shows "
![Screenshot 2024-09-28 160414](https://github.com/user-attachments/assets/e738dc19-6005-4ce2-b563-49ac29bd2490)
"
3. for gitlab, it shows sucessfully sync. but the files are not there.

**Comment by junxu-ai** [comment]
Why gitbook cannot download the markdown files directly? In "Files", only these assets files are displayed. 
I wonder why some useful functionalities are removed.

**Comment by SpeedKRNL** [comment]
I have the same problem

**Comment by macervero-abello** [comment]
I'm stuck at same point... Sync from Github to Gitbook doesn't work properly...

**Comment by 0xChicharito** [comment]
team is on Sunday, i guess

**Comment by damonYuan** [comment]
same here

**Comment by tylalin** [comment]
I have encountered the same issue since yesterday. It keeps time out whenever forced trying to sync with Github.

**Comment by addisonschultz** [comment]
Hi all - we're immediately looking into this issue, thank you for your patience!

**Comment by macervero-abello** [comment]
Problem solved!! Thank you very much, Gitbook Team!!

**Comment by EvolvingSysadmin** [comment]
The service is still timing out...

**Comment by aikoii** [comment]
Hey everyone! Thank you for your patience while our engineering team figured this one out.
​
The issue has been resolved, and Git Sync should now be fully functional again 🤞.

**If your last change before the error was made on GitBook** - meaning you had a change request that you merged and got an error from Git Sync - please create another change request with any small change in your documentation and merge it. That should export all the previous changes and restore sync.

**If your last change before the error was made on your repo** - meaning you made a commit that didn't get imported to GitBook and received an error - please create another commit with a small change in your repo. That commit should be picked up by GitBook automatically, and imported correctly alongside all the previous changes before the error and restore sync.
​
Feel free to reach out to support@gitbook.com if anything comes up.

Thank You!

**Comment by junxu-ai** [comment]
it works now. thanks!
